File: Iverson.Clients/Go/sample/models/author.go
package models

// Author is a sample entity demonstrating a one-to-many relation.
type Author struct {
	Id       string `iverson_key:"true" iverson_guid:"true"`
	Name     string
	Email    string
	Articles []string `iverson:"one_to_many:Article"`
	TenantId string   `iverson_tenant:"true"`
}

File: Iverson.Clients/TypeScript/sample/models/Author.ts
import 'reflect-metadata';
import { IversonEntity, IversonGuid, IversonKey } from '../../src/annotations.js';

@IversonEntity()
export class Author {
    @IversonKey()
    @IversonGuid()
    id: string = '';

    name: string = '';
}
